src/questions/aave-other-features.yaml

```diff
@@ -109,5 +109,169 @@
       key: C
     - content:  To borrow any amount of money for use until the user feels the need to repay the loan
       key: D
+- uuid: dbc4b555-654e-418e-a170-5e617f8aab44
+  type: MultipleChoice
+  content: Select the benefits AAVE portals
+  hint: noHint
+  explanation: Portal's feature seamlessly allows assets to flow between Aave markets on different networks, portals make alt Evm L1 chains attractive and easier to access By reducing liquidity limitations of l2-l2 bridges, the portal will allow for a seamless experience for users.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - aave-portals
+  difficultyLevel: "Medium"
+  choices:
+    - content: By reducing liquidity limitations of l2-l2 bridges, the portal will allow for a seamless experience for users
```

Comment:
Wording of this option needs to be made clearer `By reducing liquidity limitations of l2-l2 bridges, the portal will allow for a seamless experience for users`